### ISSUE TITLE
Screen reader document name doesn't match the convention

### ISSUE THREAD

**Issue by NickDJM** [coldfrontlabs/bootstrap-scss-plugins#32]
`screen-reader-breakpoints.md` should be renamed to `sr-breakpoints.md`
